test/ngResource/resourceSpec.js

```diff
@@ -102,6 +102,10 @@ describe('basic usage', function() {
     var condition = {at: '2038-01-19 03:14:08'};
 
     $httpBackend.expect('CREATE', '/fooresource', instant).respond({fid: 42});
+    $httpBackend.expectPOST('/fooresource').respond(function(method, url, data) {
```

Comment:
This is not needed here. `POST` is not a custom method (which what this test is about).